packages/generator-electrode/generators/app/templates/test/server/eslintrc

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
the entire test folder is being copied [here](https://github.com/electrode-io/electrode/blob/master/packages/generator-electrode/generators/app/index.js#L329). Then eslint files moved [here](https://github.com/electrode-io/electrode/blob/master/packages/generator-electrode/generators/app/index.js#L378)